smarts/core/agent_interface.py

```diff
@@ -193,6 +193,19 @@ class AgentsAliveDoneCriteria:
     """
 
 
+@dataclass(frozen=True)
+class ActorsAliveDoneCriteria:
+    """Require actors to persist."""
+
+    actors_of_interest: str = r""
```

Comment:
If `actors_of_interest` was a list of strings, instead of a single string, we could account for multiple vehicles of interest and also probably unify with `agents_alive` done criteria.

Reply:
This is a pattern string, so it would account based on pattern matching. I could break it up into multiple for ease of use I suppose.

Reply:
Broken up.